x-pack/test/functional/apps/lens/group3/open_in_lens/tsvb/timeseries.ts

```diff
@@ -83,5 +87,120 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       expect(await queryBar.getQueryString()).to.equal('machine.os : ios');
     });
+
+    it('should draw a reference line', async () => {
+      await visualBuilder.createNewAggSeries();
+      await visualBuilder.selectAggType('Static Value');
+      await visualBuilder.setStaticValue(10);
+
+      await header.waitUntilLoadingHasFinished();
+
+      const button = await testSubjects.find('visualizeEditInLensButton');
+      await button.click();
+      await lens.waitForVisualization('xyVisChart');
+      await retry.try(async () => {
+        const layers = await find.allByCssSelector(`[data-test-subj^="lns-layerPanel-"]`);
+
+        const referenceLineDimensions = await testSubjects.findAllDescendant(
+          'lns-dimensionTrigger',
+          layers[0]
+        );
+        expect(referenceLineDimensions).to.have.length(1);
+        expect(await referenceLineDimensions[0].getVisibleText()).to.be('Static value: 10');
+
+        const dimensions = await testSubjects.findAllDescendant('lns-dimensionTrigger', layers[1]);
+        expect(dimensions).to.have.length(2);
+        expect(await dimensions[0].getVisibleText()).to.be('@timestamp');
+        expect(await dimensions[1].getVisibleText()).to.be('Count of records');
+      });
+    });
+
+    it('should convert metric with params', async () => {
+      await visualBuilder.selectAggType('Counter Rate');
+      await visualBuilder.setFieldForAggregation('machine.ram');
+
+      await header.waitUntilLoadingHasFinished();
+
+      const button = await testSubjects.find('visualizeEditInLensButton');
+      await button.click();
+      await lens.waitForVisualization('xyVisChart');
+      await retry.try(async () => {
+        const layers = await find.allByCssSelector(`[data-test-subj^="lns-layerPanel-"]`);
+        expect(layers).to.have.length(1);
+
+        const dimensions = await testSubjects.findAll('lns-dimensionTrigger');
+        expect(dimensions).to.have.length(2);
+        expect(await dimensions[0].getVisibleText()).to.be('@timestamp');
+        expect(await dimensions[1].getVisibleText()).to.eql(
+          'Counter rate of machine.ram per second'
+        );
+      });
+    });
+
+    it('should not allow converting of not valid panel', async () => {
+      await visualBuilder.selectAggType('Counter Rate');
+      await header.waitUntilLoadingHasFinished();
+      const canEdit = await testSubjects.exists('visualizeEditInLensButton');
+      expect(canEdit).to.be(false);
+    });
+
+    it('should not allow converting of unsupported aggregations', async () => {
+      await visualBuilder.selectAggType('Variance');
```

Comment:
Out of scope for this PR, but we can convert Variance - it's this formula:
```
pow(standard_deviation(fieldName), 2)
```

Could we adjust that on a separte PR?

Reply:
Sure, we'll do that.

Reply:
No, this isn't something Lens can do I think

Reply:
@flash1293, got it, thank you